Source/ThirdParty/ANGLE/Configurations/Base.xcconfig

```diff
@@ -80,7 +80,7 @@ ANGLE_INSTALL_PATH_PREFIX = $(ANGLE_INSTALL_PATH_PREFIX_DEPLOYMENT_$(DEPLOYMENT_
 ANGLE_INSTALL_PATH_PREFIX_DEPLOYMENT_YES = $(ANGLE_INSTALL_PATH_PREFIX_DEPLOYMENT_YES_USE_ALTERNATE_$(WK_USE_ALTERNATE_FRAMEWORKS_DIR));
 ANGLE_INSTALL_PATH_PREFIX_DEPLOYMENT_YES_USE_ALTERNATE_YES = $(WK_ALTERNATE_FRAMEWORKS_DIR)/;
 
-OTHER_LDFLAGS = $(ASAN_OTHER_LDFLAGS) $(ANGLE_OTHER_LDFLAGS) $(SOURCE_VERSION_LDFLAGS);
+OTHER_LDFLAGS = $(WK_COMMON_OTHER_LDFLAGS) $(ANGLE_OTHER_LDFLAGS) $(SOURCE_VERSION_LDFLAGS);
```

Comment:
In theory, yes.

In practice, I don't want to change behavior too much on a large patch like this because there's already a good chance something will break without changing this behavior.

If we want to make these kinds of behavior changes, they should be made outside of this patch.

Also, the fact that `$(WK_COMMON_OTHER_LDFLAGS)` is used also _documents_ everywhere there might be an opportunity to switch to using `$(inherited)`, so I don't feel like we've discovered something and then didn't act on it.  We can just delay acting on it until later.

—

Now, having said that, I think it's okay to add `$(inherited)` in this specific case because it's a `Base.xcconfig` file.  However, there are other places where it's added for target xcconfig files, and I don't want to change those because they'll start inheriting values from that project's `Base.xcconfig` file, which usually means that `OTHER_LDFLAGS` will change.

Reply:
BTW, here is a perfect example of an additional change that I started to add to this patch, but I decided it would be better to break out into a separate patch in case it causes an issue and needs to be rolled out separately (it still needs a review, too):  <https://github.com/WebKit/WebKit/pull/10066>

I _almost_ broke out all the `OTHER_CPLUSPLUSFLAG` deletions into their own patch as well, but I thought that change was safe enough to include here.  (I wouldn't be opposed to breaking it out, though, if someone else thought that would be best.)